api/hiveextension/v1beta1/agentclusterinstall_types.go

```diff
@@ -171,6 +171,11 @@ type AgentClusterInstallSpec struct {
 	// Proxy defines the proxy settings used for the install config
 	// +optional
 	Proxy *Proxy `json:"proxy,omitempty"`
+
+	// PlatformType is the name for the specific platform upon which to perform the installation.
+	// +kubebuilder:validation:Enum=baremetal;vsphere;ovirt;none
```

Comment:
Can we use the same values as the openshift API? https://github.com/openshift/api/blob/9368b92/config/v1/types_infrastructure.go#L127

Reply:
Sounds like a good idea. It will require adding a conversion to assisted-service values.

Reply:
@avishayt Please also review and comment on the topic of API conversion between OpenShift's and AI's platform type names. I uploaded a code version that exposes the Openshift type for our ztp API.